src/Bicep.Core/Workspaces/SourceFileGroupingBuilder.cs

```diff
@@ -51,11 +51,11 @@ public class SourceFileGroupingBuilder
             bool forceforceModulesRestore = false)
         {
             this.fileResolver = fileResolver;
-            this.moduleDispatcher = moduleDispatcher;
+            this.dispatcher = moduleDispatcher;
             this.workspace = workspace;
             this.uriResultByArtifactReference = new(current.UriResultByArtifactReference.Select(kvp => KeyValuePair.Create(kvp.Key, kvp.Value.ToDictionary(p => p.Key, p => p.Value))));
             this.fileResultByUri = current.FileResultByUri.Where(x => x.Value.TryUnwrap() is not null).ToDictionary(x => x.Key, x => x.Value);
-            this.forceModulesRestore = forceforceModulesRestore;
+            this.forceArtifactRestore = forceforceModulesRestore;
```

Comment:
[nit] can probably shorten to just `forceRestore` if you want to cut down on the wordiness.